tembo-operator/src/config.rs

```diff
@@ -15,6 +16,12 @@ impl Default for Config {
             enable_volume_snapshot: from_env_default("ENABLE_VOLUME_SNAPSHOT", "false")
                 .parse()
                 .unwrap(),
+            volume_snapshot_rentention_period: from_env_default(
+                "VOLUME_SNAPSHOT_RETENTION_PERIOD",
```

Comment:
Can we put units on the env var name? Like `VOLUME_SNAPSHOT_RETENTION_PERIOD_HRS` or whatever the units are.

Reply:
It's in days.  Yeah good idea.  I will make that change.